extras/test-env/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# This only works if you have a single GOPATH
+#
+Vagrant.configure("2") do |config|
+  (1..4).each do |i|
+    config.vm.define "gd2test-#{i}" do |arch|
+      arch.vm.hostname = "gd2test-#{i}"
+      arch.vm.provider "docker" do |d|
+        d.image = 'gluster/glusterd2-test:latest'
+        d.name = "gd2test-#{i}"
+        d.has_ssh = true
+        d.remains_running = true
+        d.create_args = ['--privileged']
```

Comment:
In later versions of this, I'd prefer we put some thought into not running this in privileged mode, considering glusterd2 is one binary without systemd. I'm wondering if actual deployment needs glusterd2 to do setxattr on bind mounted brick paths or if there's a better way around it. It should also play nice with selinux in unprivileged mode.